newrelic/helpers.go

```diff
@@ -27,6 +27,24 @@ func parseIDs(serializedID string, count int) ([]int, error) {
 	return ids, nil
 }
 
+// Converts a hash of IDs into an array.
+// Examples: "12345:54432:66564" -> []int{12345,54432,66564}
+func parseHashedIDs(serializedID string) ([]int, error) {
```

Comment:
Perhaps a test for this one would be good.

Reply:
Agreed